connect_box/__init__.py

```diff
@@ -513,8 +526,17 @@ async def _async_initialize_valid_token(self) -> None:
                 headers=self.headers,
                 timeout=10,
             ) as response:
-                await response.text()
+                html = await response.text()
+                apiversion = re.findall("CommonAPI\.js\?v=(\d*)", html)
                 self.token = response.cookies["sessionToken"].value
+                if apiversion and len(apiversion) > 0:
+                    self.apiversion = int(apiversion[0])
+                if self.apiversion and self.apiversion >= 20220407070717:
+                    self.cookies["sessionToken"] = f"{self.token}"
+                else:
+                    self.cookies = None
+                    self.request_token_string = f"token=${self.token}&"
```

Comment:
Prefixing the value for the token with a `$` will most likely lead to login failures an router which the default firmware.